### PR TITLE
Api test cases for forgot password functional

### PR DESCRIPTION
**Pull request by ninanichik**
Test cases are checking possibility to send email with a goal of getting confirmation link.
Two test cases with valid and not valid data for positive and negative testing.

### CODE REVIEW

File: oct/tests/api/forgot_passwrd.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        assert "An email with a confirmation link has been sent your email address." in request.text
+
+    @test
+    def test_forgot_not_exist_password(self) -> None:
+        parameters = {"email": "someemailll@gmail.com"}
+        urllib3.disable_warnings()
+        request = requests.post(
+            "https://localhost/index.php?route=account/forgotten", parameters, verify=False
+        )
+        assert "The E-Mail Address was not found in our records, please try again!" in request.text
```
Comment (bambula4000) [inline oct/tests/api/forgot_passwrd.py]: Sorry, your methods names is exist / not exist password, but in assertion you checking exist / not exist email. I believe that methods name isn't good.

File: oct/tests/api/forgot_passwrd.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# pylint: disable=no-self-use # pyATS-related exclusion
+from pyats.aetest import Testcase, test
+import requests
+import urllib3
+from oct.tests import run_testcase
+
+
+class ForgotPassword(Testcase):
+    @test
+    def test_forgot_exist_password(self) -> None:
```
Comment (extsoft) [inline oct/tests/api/forgot_passwrd.py]:
```suggestion
    def test_forgot_password_for_known_user(self) -> None:
```

File: oct/tests/api/forgot_passwrd.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        parameters = {"email": "testerwom@gmail.com"}
+        urllib3.disable_warnings()
+        request = requests.post(
+            "https://localhost/index.php?route=account/forgotten", parameters, verify=False
+        )
+        assert "An email with a confirmation link has been sent your email address." in request.text
+
+    @test
+    def test_forgot_not_exist_password(self) -> None:
```
Comment (extsoft) [inline oct/tests/api/forgot_passwrd.py]:
```suggestion
    def test_forgot_password_for_unknown_user(self) -> None:
```

Comment (bambula4000) [reply oct/tests/api/forgot_passwrd.py]: Now it looks much better! Thanks

File: oct/tests/api/forgot_passwrd.py (hunk 2 above)
Comment (bambula4000) [reply oct/tests/api/forgot_passwrd.py]: Now it looks much better! Thanks

**Comment by bambula4000** [comment]
@ninanichik 
And please, don't forget - commit subject to 50 characters or less, description to 72 characters in line.
And subject should begin with a imperative verb, like - Forgot password functional test cases via API, etc.

**Comment by bambula4000** [comment]
And please, don't forget - commit subject to 50 characters or less, description to 72 characters in line.
And subject should begin with a imperative verb, like - Forget password functional test cases via API, etc.

**Pull request merged**